cranelift/codegen/src/isa/x64/inst/mod.rs

```diff
@@ -228,14 +229,24 @@ impl Inst {
         dst_remainder: WritableGpr,
     ) -> Inst {
         divisor.assert_regclass_is(RegClass::Int);
-        Inst::Div {
-            size,
-            signed,
-            divisor: GprMem::new(divisor).unwrap(),
-            dividend_lo,
-            dividend_hi,
-            dst_quotient,
-            dst_remainder,
+        // TODO
```

Comment:
Want to fill out this TODO a little?

Reply:
Ah yeah this ended up going away in a later commit (was sort of figuring out my way through things as I went along)